src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1884,8 +1887,55 @@ public void testLabels() throws DockerException, InterruptedException {
     sut.startContainer(id);
 
     ContainerInfo containerInfo = sut.inspectContainer(id);
-    assertThat(containerInfo.config().labels().keySet(), contains("name", "version"));
-    assertThat(containerInfo.config().labels().values(), contains("starship", "1.6.2"));
+    assertThat(containerInfo.config().labels().keySet(), containsInAnyOrder("name", "foo"));
+    assertThat(containerInfo.config().labels().values(), containsInAnyOrder("starship", "bar"));
```

Comment:
why not assert the mapping between the key and value instead?

    assertThat(containerInfo.config().labels(), hasEntry("name", "starship"));
    assertThat(containerInfo.config().labels(), hasEntry("foo", "bar"));

or a simple equality test

    assertThat(containerInfo.config().labels(), is(labels));